README.md

```diff
@@ -291,6 +291,111 @@ do this on [my personal site](https://github.com/kentcdodds/kentcdodds.com)
 which is generated at build-time on the server. Learn about rendering
 [`react` on the server][react-ssr] and [`glamor` too][glamor-ssr].
 
+### Example Style Objects
+
+Style objects can affect pseudo-classes and pseduo-elements, complex CSS
+selectors, introduce keyframe animations, and use media queries:
+
+<details>
+  <summary>pseudo-class</summary>
+  ```javascript
```

Comment:
Sorry! I should have mentioned this. I've seen this in the past. You need to add a line of whitespace after `<summary>` otherwise it renders really wonky:

<img width="584" alt="screen shot 2017-04-05 at 4 26 28 pm" src="https://cloud.githubusercontent.com/assets/1500684/24729706/be6c7c48-1a1c-11e7-9482-4135673ca156.png">